code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2611,3 +2611,41 @@
 		var/color
 		CONVERT_PH_TO_COLOR(exposed_atom.reagents.ph, color)
 		exposed_atom.add_atom_colour(color, WASHABLE_COLOUR_PRIORITY)
+
+/datum/reagent/ants
+	name = "Ants"
+	description = "A sample of a lost breed of Space Ants (formicidae bastardium tyrannus), they are well-known for ravaging the living shit out of pretty much anything."
+	reagent_state = SOLID
+	color = "#993333"
+	taste_mult = 1.3
+	taste_description = "<span class='warning'>ANTS OH GOD</span>"
```

Comment:
Maybe a little bit tamer? something like "a tickling in the back of your throat"?

Reply:
"You taste scuttling going down your throat" doesn't sound like a very clear statement, it sounds more like you're describing touch rather than taste. While I personally enjoy the screaming realization that you just drank ants, I _will_ change it if you feel that it's too extreme. I'll do that now, then.

Reply:
If you think this adds more flavor feel free to stick with it, it's not harmful on the large scale